common/src/webida/plugins/editors/workbench-panels.js

```diff
@@ -54,6 +54,8 @@ define([
         return docFrag;
     }
 
+    var QUIT = 'Quit';
```

Comment:
This constant should be used also to the following line:
createDialog(modifiedFileNames.join(', '), 'Quit', action, cancel);